src/Web/Sfa.Das.Sas.ApplicationServices/Handlers/GetStandardHandler.cs

```diff
@@ -27,20 +34,32 @@ public GetStandardResponse Handle(GetStandardQuery message)
             if (intId < 0)
             {
                 response.StatusCode = GetStandardResponse.ResponseCodes.InvalidStandardId;
-
                 return response;
             }
 
             if (standard == null)
             {
                 response.StatusCode = GetStandardResponse.ResponseCodes.StandardNotFound;
-
                 return response;
             }
 
             response.Standard = standard;
             response.SearchTerms = message.Keywords;
 
+            try
+            {
+                var assessmentOrganisations = _getAssessmentOrgs.ByStandard(standard.StandardId);
+                response.AssessmentOrganisations = assessmentOrganisations?.ToList() ?? new List<Organisation>();
+            }
+            catch (EntityNotFoundException)
+            {
+                response.StatusCode = GetStandardResponse.ResponseCodes.AssessmentOrgsEntityNotFound;
+            }
```

Comment:
we should assign an empty list of organisations after catching EntityNotFoundException `response.AssessmentOrganisations = new List<Organisation>();`